src/main/msp/msp_protocol.h

```diff
@@ -62,7 +62,7 @@
 #define MSP_PROTOCOL_VERSION                0
 
 #define API_VERSION_MAJOR                   1  // increment when major changes are made
-#define API_VERSION_MINOR                   47 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
+#define API_VERSION_MINOR                   48 // increment after a release, to set the version for all changes to go into the following release (if no changes to MSP are made between the releases, this can be reverted before the release)
```

Comment:
is this necessary? IMO new uart range is backward compatible

Reply:
It is so the configurator can identify when to apply the new index or not

Reply:
I didn't study configurator much. It seems that only source of ` FC.SERIAL_CONFIG.ports[].identifier` is BF, configurator is using this value only to display friendly name. So all that should be necessary is to add new mapping (50: UART0, ... ) to portIdentifierToNameMapping. Configurator will use new IDs without other changes

Also, per-port MSP structure is easily extensible. Maybe we can add post name to MSP - that way configurator will accept any ports (only VCP is partially hardcoded to implement safety warning). Board can even name ports in future

Reply:
I'll need to test that theory, in case name to identifier is used anywhere in configurator, if not then it'll be fine - and just another named mapping entry needed for the values 50+.